pages/login.tsx

```diff
@@ -43,6 +55,10 @@ export default function Login(): JSX.Element {
                         fontSize="10px"
                         fontWeight="400"
                         mt="20px"
+                        onClick={() => {
+                            setLocalStorageEmail(email);
+                            signIn("email", { email: email });
```

Comment:
There's a bit of buffer time between when you press the "Continue" button and when it actually redirects - would it be easy to add some sort of loading circle to show that something to show that the button worked and it's just loading? @stacykwok @christinaruu thoughts?

If we go with that, I think you can do something like `const [session, loading] = useSession()` and then if `loading` is true, then you render some sort of loading circle or something like that.

Reply:
Yess, I think adding a loading screen would be a great idea! Is there a default loading icon or do we need to make/find one?

Reply:
Chakra has some spinners, you can see them here: https://chakra-ui.com/docs/feedback/spinner

Reply:
Nvm - it looks like it got added
![image](https://user-images.githubusercontent.com/42329517/126842613-95258735-46e2-4958-a5a5-3dc90753f327.png)